core/tests/test-modules-mappings/acceptance/curriculum-admin/create-edit-and-delete-classroom.txt

```diff
@@ -0,0 +1,6 @@
+core/templates/pages/classroom-admin-page/classroom-admin-page.module.ts
+core/templates/pages/classroom-page/classroom-page.module.ts
+core/templates/pages/library-page/library-page.module.ts
```

Comment:
Remove one of the library-page.module.ts.

Reply:
Done.